push_notifications/wns.py

```diff
@@ -0,0 +1,359 @@
+"""
+Windows Notification Service
+
+Documentation is available on the Windows Dev Center:
+https://msdn.microsoft.com/en-us/windows/uwp/controls-and-patterns/
+tiles-and-notifications-windows-push-notification-services--wns--overview
+"""
+
+import json
+import xml.etree.ElementTree as ET
+from xml.etree.ElementTree import tostring
+
+try:
+	from urllib.error import HTTPError
+	from urllib.request import Request, urlopen
+except ImportError:
+	# Python 2 support
+	from urllib2 import HTTPError, Request, urlopen
+
+from django.core.exceptions import ImproperlyConfigured
+
+from . import NotificationError
+from .settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
+WNS_ACCESS_URL = "https://login.live.com/accesstoken.srf"
```

Comment:
I'm fine with this either way, but what is the reason for making this configurable at the project level? 
It would seem that if WNS changes the url or api in any way it would have to be maintained by the app(you) and wouldn't be something the project developer would want/need to have knowledge about. 

Basically, If it needs to be overridden, shouldn't it be a bug that needs fixing in master?